src/graphnet/models/task/task.py

```diff
@@ -76,14 +92,22 @@ def __init__(
         """
         # Base class constructor
         super().__init__()
-
         # Check(s)
+        if target_labels is None:
+            target_labels = self.default_target_labels
         if isinstance(target_labels, str):
             target_labels = [target_labels]
-
+        if prediction_labels is not None:
+            if isinstance(prediction_labels, str):
+                prediction_labels = [prediction_labels]
+        else:
+            prediction_labels = self.default_prediction_labels
+        assert isinstance(target_labels, List)  # mypy
+        assert isinstance(prediction_labels, List)  # mypy
```

Comment:
To keep the structure consistent with `target_labels` above.
```suggestion

        if prediction_labels is None:
            prediction_labels = self.default_prediction_labels
        if isinstance(prediction_labels, str):
            prediction_labels = [prediction_labels]
        
        assert isinstance(target_labels, List)  # mypy
        assert isinstance(prediction_labels, List)  # mypy
```